specification/monitor/resource-manager/readme.md

```diff
@@ -28,14 +28,23 @@ These are the global settings for the MonitorClient API.
 title: MonitorClient
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2019-06
+tag: package-preview-2020-05
```

Comment:
I don't think we should change this one.

Reply:
@MyronFanQiu  
I used Microsoft [OpenApiHub](https://portal.azure-devex-tools.com) and it committed 27d3d19af, 847a13e8b, 2a9e0ce87  which include auto generated changes to the readme file. I fixed this line and all other by using the version from the private repository (which I changed manually). Please review.